frontend/src/components/Cases/AccordionSvg.js

```diff
@@ -0,0 +1,44 @@
+import Svg from "../../assets/svg/Svg";
+
+const AccordionSvg = ({ testResult }) => {
+  if (testResult === "skipped") {
```

Comment:
Maybe `switch-case` can be used instead?

Reply:
Done